src/java.sql.rowset/share/classes/com/sun/rowset/RowSetResourceBundle.properties

```diff
@@ -123,15 +123,15 @@ crsreader.caldetected = Detected a Calendar
 #CachedRowSetWriter exceptions
 crswriter.connect = Unable to get connection
 crswriter.tname = writeData cannot determine table name
-crswriter.params1 = Value of params1 : {0} 
-crswriter.params2 = Value of params2 : {0} 
-crswriter.conflictsno =  conflicts while synchronizing 
+crswriter.params1 = Value of params1 : {0}\u0020
+crswriter.params2 = Value of params2 : {0}\u0020
+crswriter.conflictsno =  conflicts while synchronizing\u0020
```

Comment:
this is tricky.  if this value is a part of a sentence (i.e. something like "5 conflicts..."), the localization is likely to be wrong.  it's hard to tell without looking further into the code.